.travis.yml

```diff
@@ -20,8 +22,6 @@ env:
 matrix:
   allow_failures:
     - env: RUN=test/update-coveralls
-    # Temporarily allow the integration tests to fail until we fix them
-    - env: RUN="make test-integration"
```

Comment:
Sorry. I just rebased off master and pushed again. Take another look here. Thanks!